ui/Screen/UserProfile/UserProfileHeader.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="typescript">
+  import type { Avatar as AvatarT } from "../../src/proxy/identity";
+
+  import { Avatar } from "../../DesignSystem/Primitive";
+  import { PeerId } from "../../DesignSystem/Component";
+
+  export let name: string;
+  export let peerId: string;
+
+  export let avatarFallback: AvatarT | undefined = undefined;
```

Comment:
Can you check if this property is always provided and whether we can remove the `undefined` case?

Reply:
As far as I can see, Identity always has the `avatarFallback`: 738f4194332d1f45366883dd2fd42cff81d15eed.